deeprank/learn/NeuralNet.py

```diff
@@ -494,7 +495,10 @@ def load_nn_params(self):
         """Get NeuralNet parameters from a saved model."""
         self.task = self.state['task']
         self.criterion = self.state['criterion']
-        self.hit_cutoff = self.state['hit_cutoff']
+        try:
+            self.hit_cutoff = self.state['hit_cutoff']
+        except Exception:
+            print('No hit_cutoff provided')
```

Comment:
Please use `logger` to output messages. 
I guess this `Exception` only applies to our published models. If so, we have to tell users what to do in the message, like `No "hit_cutoff" found in {model}. Please set it in function when doing prediction"`.

Reply:
The hit cutoff is used only if target values are provided. In prediction mode you do not need it, you need it for benchmark only.